homeassistant/components/volvooncall.py

```diff
@@ -123,6 +126,13 @@ def __init__(self, hass, vin, attribute):
         self._state.entities[self._vin].append(self)
 
     @property
+    def entity_id(self):
```

Comment:
Yes. You can move the property into entity class inside `sensor/volvooncall.py` and use the DOMAIN from `sensor/__init__.py` or you remove the `entity_id` property and use a unique Name that generate a unique entity_id inside core.